vercel.json

```diff
@@ -2,7 +2,7 @@
   "functions": {
     "api/*.js": {
       "memory": 128,
-      "maxDuration": 10
+      "maxDuration": 30
```

Comment:
Edited the **"maxDuration"** from 10 to 30 **"maxDuration": 30**, so You don't have to worry about anything when merging

Reply:
@MitkoVtori Thanks! We need 30 for the public instance (see https://github.com/anuraghazra/github-readme-stats/issues/1416).